019-restruture-the-installation-files.md

```diff
@@ -0,0 +1,64 @@
+# Restructure the installation files
+
+This proposal suggests restructuring to the folders of the [`strimzi-kafka-operator` GitHub repository](https://github.com/strimzi/strimzi-kafka-operator).
+
+## Current situation
+
+Currently, the Strimzi installation files in our master branch correspond to the images built from the master branch.
+That means they are under constant development and often work only with the freshly pulled images.
+That suits well for development work, but not necessarily for the users.
+They often checkout the GitHub repository, see the install or example folders and try them, not realising that they're using `master` rather than a stable release.
+If they just use them for a short time it should not matter.
+But if they keep using them, they will run into issues later.
+
+## Motivation
+
+Improve the experience of the users who checkout Strimzi and want to install / try it.
+
+## Proposal
+
+We should change the way we manage these files.
+This proposal suggests following:
+* Create a new subdirectory `packaging`
+    * This subdirectory will contain the _in-development_ versions of `install`, `examples` and `helm-chart` directories.
```

Comment:
So the release zips will pretty much consist of the contents of this directory, right?

Reply:
I do not expect the release artifacts to change from how they look today. They will just be built based on the files from `packaging` and not from the project _root_. I can make it more clear in the proposal.